packages/flutter_tools/test/general.shard/generate_localizations_test.dart

```diff
@@ -2155,7 +2182,7 @@ import 'output-localization-file_en.dart' deferred as output-localization-file_e
         "description": "The first day of spring",
         "placeholders": {
           "springStartDate": {
-            "format": "MMMMd"
+            "format": "MMMd"
```

Comment:
Is there any particular reason this was changed? If not, we should probably change it back.

Reply:
No, I thought I was fixing a typo, because I thought this was defining the input shape and not how the placeholder will be rendered into the string